src/gemm_v2.cpp

```diff
@@ -121,6 +121,34 @@ std::ostream& operator<<(std::ostream& stream, const GemmDescriptor& gemm_desc)
                   << "dataType " << gemm_desc.dataType << "} ";
 }
 
+#if MIOPEN_USE_ROCBLAS
+
+inline rocblas_atomics_mode DisableRocblasAtomics(const miopen::Handle& handle)
+{
+    MIOPEN_LOG_I2("");
+    rocblas_atomics_mode cur_mode;
+    rocblas_status status = rocblas_get_atomics_mode(handle.rhandle().get(), &cur_mode);
+    assert(status == rocblas_status::rocblas_status_success);
+    (void)status; // WA till C++17 [[maybe_unused]]
+    if(cur_mode == rocblas_atomics_allowed)
+    {
+        status = rocblas_set_atomics_mode(handle.rhandle().get(), rocblas_atomics_not_allowed);
+        assert(status == rocblas_status::rocblas_status_success);
+        (void)status; // WA till C++17 [[maybe_unused]]
+    }
+    return cur_mode;
+}
+
+inline void SetRocblasAtomics(const miopen::Handle& handle, rocblas_atomics_mode mode)
+{
+    MIOPEN_LOG_I2("");
+    rocblas_status status = rocblas_set_atomics_mode(handle.rhandle().get(), mode);
+    assert(status == rocblas_status::rocblas_status_success);
+    (void)status; // WA till C++17 [[maybe_unused]]
```

Comment:
[Note] If WG21 continue to litter the language with syntactic sugar, then termites will come and gobble everything 🐜 

BTW `std::ignore = status` can be used instead of `(void)`, I guess (`std::ignore` is defined in `<tuple>` IIRC)